android-components/components/browser/menu/src/main/res/values/strings.xml

```diff
@@ -13,4 +13,6 @@
     <string name="mozac_browser_menu_addons_manager">Add-ons Manager</string>
     <!-- Content description for the action bar "up" button -->
     <string name="action_bar_up_description">Navigate up</string>
+    <!-- Content description for the action bar "up" button of the add-ons sub menu item -->
```

Comment:
I know you're copying from the button above, but is "up" the right choice here? To me, it's "back".

"Navigate up" implies more scrolling to the top of something.

Reply:
I don't know why it is "up" here, maybe because we go one level up in the menu tree? (from sub-menu to main menu)

Reply:
Do you think we should change that? Ideally in all the strings defined in this file?

Reply:
I would ask accessibility folks. If that works for them, then we should keep it as is.

Reply:
I talked to @anna-yeddi and "back" is clearer so I updated.